rllib/models/modelv2.py

```diff
@@ -223,6 +232,26 @@ def from_batch(self, train_batch, is_training=True):
             i += 1
         return self.__call__(input_dict, states, train_batch.get("seq_lens"))
 
+    def get_view_requirements(self, is_training=False):
+        """Returns a list of ViewRequirements for this Model (or None).
+
+        A ViewRequirement object tells the caller of this Model, which
+        data at which timesteps are needed by this Model. This could be a
+        sequence of past observations, internal-states, previous rewards, or
+        other episode data/previous model outputs.
+
+        Args:
+            is_training (bool): Whether the returned requirements are for
+                training or inference (default).
+
+        Returns:
+            Optional[Dict[str, Dict]]: The view requirements as a dict mapping
+                column names e.g. "obs" to config dicts containing supported
+                fields.
+                TODO: (sven) Currently only `timesteps==0` can be setup.
+        """
+        return None
```

Comment:
Could this return a default config instead of None? Then it doesn't need to be optional type either.

Reply:
done

Reply:
done

Reply:
done